### ISSUE TITLE
Android Auto - How to add skip/previous button

### ISSUE THREAD

**Issue by jqi26** [androidx/media#1778]
I want to add next and previous buttons to jump to different media items in the original list of browsable media items. From what I understand the `Player.COMMAND_SEEK_TO_NEXT_MEDIA_ITEM` command needs to be available for this functionality to work. In the `onConnect` override in my `MediaLibrarySession.Callback` I do not call `setAvailablePlayerCommands`, so I'm assuming it should include the `Player.COMMAND_SEEK_TO_NEXT_MEDIA_ITEM` command already (I have also tried `setAvailablePlayerCommands(ConnectionResult.DEFAULT_PLAYER_COMMANDS)`). 

However, the buttons still do not appear. They also don't appear on the notification player on the phone. In `onSetMediaItems`/`onAddMediaItems` I can see `mediaSession.player.isCommandAvailable(Player.COMMAND_SEEK_TO_NEXT_MEDIA_ITEM)` is false. Any idea what I'm doing wrong?

**Comment by marcbaechinger** [comment]
Thanks for your report. 

I agree your expectations are correct.

> They also don't appear on the notification player on the phone.

You should always see a back button on the notification, not matter what I think. 

Can you please provide us with or confirm some information:

- please verify you have more than a single item in the playlist. The next command is only available when there is a next item.
- when your app is running and the session is up, can you please do a `adb shell dumpsys media_session` and post the output here?
- Can you repro the same behaviour with the [session demo app](https://github.com/androidx/media/tree/release/demos/session) (Android Auto) or the s[ession demo Automotive OS app](https://github.com/androidx/media/tree/release/demos/session_automotive)? This will help us to rule out app isssue. I'd say that works with these to demo apps, but I'm fine in you educating me it doesn't for some configurations.
- I assume you mean Android Auto and not Automotive OS. Is this correct? It doesn't really matter I think but when we have to dig it would be good to know we are talking aboutthe same.

**Comment by jqi26** [comment]
Thanks @marcbaechinger. Looks like it was a dumb mistake on my part. I took another look at the media3 sample and saw `DemoMediaLibrarySessionCallback.onSetMediaItems` makes a call to `maybeExpandSingleItemToPlaylist` to get additional items for the playlist. This prompted me to double-check our implementation of `onSetMediaItems` noticed a bug in my changes that wasn't invoking our equivalent of `maybeExpandSingleItemToPlaylist`. I've fixed and the buttons and showing up now as expected.
